src/devices/video/vrender0.cpp

```diff
@@ -650,16 +575,13 @@ int vr0video_device::vrender0_ProcessPacket(uint32_t PacketPtr)
 		if (Packet0 & 0x8)  //Texture Enable
 		{
 			Quad.u.Imageb = TEXTURE + 128 * m_RenderState.FontOffset;
-			Quad.Tile = (uint16_t*) (TEXTURE + 128 * m_RenderState.TileOffset);
+			Quad.Tile = (u16*) (TEXTURE + 128 * m_RenderState.TileOffset);
 			if (!m_RenderState.PixelFormat)
 				Quad.Pal = m_InternalPalette + (m_RenderState.PaletteBank * 16);
 			else
 				Quad.Pal = m_InternalPalette;
 
-			if (m_RenderState.TextureMode)   //Tiled
-				DrawTile[m_RenderState.PixelFormat + 4 * Mode](&Quad);
-			else
-				DrawImage[m_RenderState.PixelFormat + 4 * Mode](&Quad);
+			DrawQuad(bpp[m_RenderState.PixelFormat], m_RenderState.TextureMode, &Quad);
```

Comment:
`DrawQuad()` is misleading: it doesn't follow the documentation naming conventions.